File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.MoreEx_PB-JS-Nested-Conditional-Statements/P05_Vacation.js
function vacation(input) {
    let budget = Number(input[0]);
    let seasonType = input[1];

    let accommodation = "";
    let destination = "";
    let holidayCost = 0.0;

    if (budget > 3000) {
        accommodation = "Hotel";
        if (seasonType === "Summer") {
            destination = "Alaska";
        } else {
            destination = "Morocco";
        }
        holidayCost = budget * 0.9;
    } else
        if (budget > 1000) {
            accommodation = "Hut";
            if (seasonType === "Summer") {
                destination = "Alaska";
                holidayCost = budget * 0.8;
            } else {
                destination = "Morocco";
                holidayCost = budget * 0.6;
            }
        } else {
            accommodation = "Camp";
            if (seasonType === "Summer") {
                destination = "Alaska";
                holidayCost = budget * 0.65;
            } else {
                destination = "Morocco";
                holidayCost = budget * 0.45;
            }
        }
    console.log(`${destination} - ${accommodation} - ${holidayCost.toFixed(2)}`);
}

vacation([800,
    "Summer"
])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P03AdditionalProblems_ConditionalStatementsAdvanced/P05_Vacation.java
package P03AdditionalProblems_ConditionalStatementsAdvanced;

import java.util.Scanner;

public class P05_Vacation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double budget = Double.parseDouble(scanner.nextLine());
        String season = scanner.nextLine();
        String accommodationType = "";
        String destination = "";

        if (budget <= 1000) {
            accommodationType = "Camp";
            if (season.equals("Summer")) {
                destination = "Alaska";
                budget = budget * 0.65;
            } else {
                destination = "Morocco";
                budget = budget * 0.45;
            }
        } else if (budget <= 3000) {
            accommodationType = "Hut";
            if (season.equals("Summer")) {
                destination = "Alaska";
                budget = budget * 0.8;
            } else {
                destination = "Morocco";
                budget = budget * 0.6;
            }
        } else {
            accommodationType = "Hotel";
            budget = budget * 0.9;
            if (season.equals("Summer")) {
                destination = "Alaska";
            } else {
                destination = "Morocco";
            }
        }
        System.out.printf("%s - %s - %.2f", destination, accommodationType, budget);
    }

}

